content/applications/productivity/iot/devices/screen.rst

```diff
@@ -66,27 +69,27 @@ Things)` Box homepage.
 Usage
 =====
 
-Show Point of Sales orders to customers
----------------------------------------
+Show Point of Sale orders to customers
+--------------------------------------
 
-To use the screen in the :guilabel:`Point of Sale` app, go to :menuselection:`Point of Sale -->
-Configuration --> Point of Sale`, open the desired screen, click on :guilabel:`Edit`, and enable the
+To use the screen in the :menuselection:`Point of Sale` app, go to :menuselection:`Point of Sale -->
+Configuration --> Point of Sale`, open the desired screen, :guilabel:`Edit`, and enable the
 :guilabel:`IoT Box Devices` feature.
 
 Now, select :guilabel:`IoT Box` in the drop-down menu and choose the screen in the
-:guilabel:`Customer Display` option. Then, click :guilabel:`Save`.
+:guilabel:`Customer Display` option. Then, :guilabel:`Save` if required.
 
-    .. image:: screen/screen-pos-screen-config.png
-       :align: center
-       :alt: Connect the screen display to the Point of Sale app.
+.. image:: screen/screen-pos-screen-config.png
+   :align: center
+   :alt: Connect the screen display to the Point of Sale app.
 
-The screen is now available for :abbr:`PoS (Point of Sale)` sessions. A "screen" icon will appear
-in the top bar to inform the connection status with the screen.
+The screen is now available for :abbr:`PoS (Point of Sale)` sessions. A :guilabel:`Screen` icon will
```

Comment:
We don't need a guilabel here since the word "Screen" isn't part of the UI.
```suggestion
The screen is now available for :abbr:`PoS (Point of Sale)` sessions. A screen icon will
```